packages/integration_test/README.md

```diff
@@ -16,256 +16,3 @@ dev_dependencies:
 
 For the latest documentation, see [Integration
 testing](https://flutter.dev/docs/testing/integration-tests).
-
```

Comment:
None of the code for this package exists any more, so the only thing the readme is still here for is a pointer in case of stale external links to this package's source. The README included things like links to `example/` files that don't even exist any more, so I just removed it all. (This is consistent with the current README state in `e2e/`, which is the even-older location of this package.)